expression/builtin_info_vec_test.go

```diff
@@ -88,6 +89,10 @@ var vecBuiltinInfoCases = map[string][]vecExprBenchCase{
 		{retEvalType: types.ETInt, childrenTypes: []types.EvalType{}},
 		{retEvalType: types.ETInt, childrenTypes: []types.EvalType{types.ETInt}},
 	},
+	ast.Benchmark: {
```

Comment:
Thanks for your contribution. Could you please add more test cases?

Reply:
Comment addressed, PTAL, thanks.

Reply:
```
> GO111MODULE=on go test -check.f TestVectorizedBuiltinInfoFunc
PASS: builtin_info_vec_test.go:112: testEvaluatorSuite.TestVectorizedBuiltinInfoFunc	0.054s
OK: 1 passed
PASS
ok  	github.com/pingcap/tidb/expression	0.080s
```

```
> go test -v -benchmem -bench=BenchmarkVectorizedBuiltinInfoFunc -run=BenchmarkVectorizedBuiltinInfoFunc -args builtinBenchmarkSig
goos: darwin
goarch: amd64
pkg: github.com/pingcap/tidb/expression
BenchmarkVectorizedBuiltinInfoFunc/builtinBenchmarkSig-VecBuiltinFunc-12         	  997416	      1153 ns/op	       0 B/op	       0 allocs/op
BenchmarkVectorizedBuiltinInfoFunc/builtinBenchmarkSig-NonVecBuiltinFunc-12      	   19207	     62892 ns/op	       0 B/op	       0 allocs/op
BenchmarkVectorizedBuiltinInfoFunc/builtinBenchmarkSig-VecBuiltinFunc#01-12      	 1000000	      1228 ns/op	       0 B/op	       0 allocs/op
BenchmarkVectorizedBuiltinInfoFunc/builtinBenchmarkSig-NonVecBuiltinFunc#01-12   	   17964	     66991 ns/op	       0 B/op	       0 allocs/op
BenchmarkVectorizedBuiltinInfoFunc/builtinBenchmarkSig-VecBuiltinFunc#02-12      	  118618	      9052 ns/op	       0 B/op	       0 allocs/op
BenchmarkVectorizedBuiltinInfoFunc/builtinBenchmarkSig-NonVecBuiltinFunc#02-12   	   17176	     68769 ns/op	       0 B/op	       0 allocs/op
BenchmarkVectorizedBuiltinInfoFunc/builtinBenchmarkSig-VecBuiltinFunc#03-12      	  217189	      5550 ns/op	       0 B/op	       0 allocs/op
BenchmarkVectorizedBuiltinInfoFunc/builtinBenchmarkSig-NonVecBuiltinFunc#03-12   	    9430	    130604 ns/op	       0 B/op	       0 allocs/op
BenchmarkVectorizedBuiltinInfoFunc/builtinBenchmarkSig-VecBuiltinFunc#04-12      	  197772	      5937 ns/op	       0 B/op	       0 allocs/op
BenchmarkVectorizedBuiltinInfoFunc/builtinBenchmarkSig-NonVecBuiltinFunc#04-12   	   10000	    130771 ns/op	       0 B/op	       0 allocs/op
BenchmarkVectorizedBuiltinInfoFunc/builtinBenchmarkSig-VecBuiltinFunc#05-12      	  162732	      6484 ns/op	       0 B/op	       0 allocs/op
BenchmarkVectorizedBuiltinInfoFunc/builtinBenchmarkSig-NonVecBuiltinFunc#05-12   	    8922	    145922 ns/op	       0 B/op	       0 allocs/op
BenchmarkVectorizedBuiltinInfoFunc/builtinBenchmarkSig-VecBuiltinFunc#06-12      	  681084	      1795 ns/op	       0 B/op	       0 allocs/op
BenchmarkVectorizedBuiltinInfoFunc/builtinBenchmarkSig-NonVecBuiltinFunc#06-12   	   12678	     95094 ns/op	       0 B/op	       0 allocs/op
BenchmarkVectorizedBuiltinInfoFunc/builtinBenchmarkSig-VecBuiltinFunc#07-12      	   97280	     12176 ns/op	       0 B/op	       0 allocs/op
BenchmarkVectorizedBuiltinInfoFunc/builtinBenchmarkSig-NonVecBuiltinFunc#07-12   	    9666	    117800 ns/op	       0 B/op	       0 allocs/op
PASS
ok  	github.com/pingcap/tidb/expression	24.269s
```